Dev/config/strings.js

```diff
@@ -136,11 +136,13 @@ export default {
     ThereIsAlreadyAStudentWithThatName: 'There is already a student with that name',
     EnterYourStudentsName: 'Enter your student\'s name',
     Whoops: 'Whoops',
-    AddStudents: 'Add Students',
     JoinMyClass: 'Join my class on Quran Connect. Here is the class code: \n',
     AddYourStudents: 'Add Your Students',
-    Or: 'Or',
-    AddStudentsManually: 'Manually Add Your Students',
+    Or: 'Or: ',
+    AddStudentsManually: 'Add students manually',
+    AddStudents: 'Add Students',
+    AddManualStudents: 'Add Manual Students',
```

Comment:
I think Add Students Manually is a better string. Also, is this a title? should be be Add students manually instead?